app.js

```diff
@@ -17,15 +22,19 @@ app.get('/api/articles/', getArticles)
 
 app.get('/api/articles/:article_id', getArticlesById);
 
+app.get('/api/articles/:article_id/comments', getAllCommentsForAnArticle)
+
+
+//app.post('/api/articles/:article_id/comments', insertCommentsForAnArticle);
 
 
 app.use((err, req, res, next) => {
   console.error(err.stack);
   res.status(500).json({ error: "internal server error" });
 });
 
-app.listen(4000, () => {
-  console.log("server running on port 4000");
-});
+// app.listen(4000, () => {
+//   console.log("server running on port 4000");
+// });
```

Comment:
We're using supertest to connect to our server instead of making it listen and connecting to it directly, so we shouldn't have an app.listen in the app.js